zipkin-ui/templates/trace.mustache

```diff
@@ -4,13 +4,14 @@
     <li class=''><a href='#'><strong>Services:</strong> <span class='badge'>{{services}}</span></a></li>
     <li class=''><a href='#'><strong>Depth:</strong> <span class='badge'>{{depth}}</span></a></li>
     <li class=''><a href='#'><strong>Total Spans:</strong> <span class='badge'>{{totalSpans}}</span></a></li>
+    <li class='navbar-right'><a href='/api/v1/trace/{{ traceId }}' id='traceJsonLink'><button type="button" class="btn btn-primary btn-xs badge" value="uiShowJson">JSON</button></a></li>
```

Comment:
Hm, can you use the `value` thing on an `<a>` element? Using the same event mechanism as everything else is convenient.